execution/sym_region_frag_machine.ml

```diff
@@ -1253,22 +1272,44 @@ struct
 	else 
 	  match self#decide_wd "Load" off_exp cloc with
 	    | None -> None
-	    | Some wd -> self#table_load cloc off_exp wd ty
+	    | Some wd -> self#table_load cloc (Some 0) off_exp wd ty
 	    
     method private handle_load addr_e ty =
       if !opt_trace_offset_limit then
 	Printf.printf "Loading from... %s\n" (V.exp_to_string addr_e);
       match self#maybe_table_or_concrete_load addr_e ty with
-        | Some v -> (v, ty)
-        | None ->
-      let (r, addr) = self#eval_addr_exp_region addr_e 0x8000 in
-      let v =
-	(match ty with
-	   | V.REG_8  -> form_man#simplify8  (self#load_byte_region  r addr)
-	   | V.REG_16 -> form_man#simplify16 (self#load_short_region r addr)
-	   | V.REG_32 -> form_man#simplify32 (self#load_word_region  r addr)
-	   | V.REG_64 -> form_man#simplify64 (self#load_long_region  r addr)
-	   | _ -> failwith "Unsupported memory type") in
+      | Some v -> (v, ty)
+      | None ->
+	let (r, addr, is_table_load, off_expr, cloc ) = 
+	  self#eval_addr_exp_region addr_e 0x8000 in
+	let sym_region_table_v = 
+	  if is_table_load = true then (
+	    if addr <> 0L then 
+	      failwith "table load from symbolic region with non-zero offset, panic!";
+	    match self#decide_wd "Load" off_expr 0L with
+	    | None -> None
+	    | Some wd -> 
+	      Printf.printf 
```

Comment:
If this printf will likely be useful in the future, it should be controlled by a tracing option, such as opt_trace_tables. Otherwise probably remove it.